tests/TestCase/Database/ExpressionTypeCastingIntegrationTest.php

```diff
@@ -136,14 +136,12 @@ public function testSelectWithBetween(): void
     {
         $this->_insert();
         $result = $this->connection->selectQuery(fields: 'id', table: 'ordered_uuid_items')
-            ->where(function (QueryExpression $exp) {
-                return $exp->between(
-                    'id',
-                    '482b7756-8da0-419a-b21f-27da40cf8569',
-                    '48298a29-81c0-4c26-a7fb-413140cf8569',
-                    'ordered_uuid'
-                );
-            })
+            ->where(fn (QueryExpression $exp)=> $exp->between(
```

Comment:
@ADmad Did we come up with a standard for arrow functions? They should have spaces before => ?

Reply:
We can lean on the PER standard (successor to PSR12 from which our formatting rules are derived) https://www.php-fig.org/per/coding-style/#71-short-closures. However, it looks like formatting for arrow functions is un-implemented in PHPCS https://github.com/squizlabs/PHP_CodeSniffer/issues/3321. I could implement it in our coding standards, and then we can do a simple batch update.

Reply:
`slevomat/coding-standard` does have a sniff for arrow functions, I have created a PR for our code sniffer to use it.